cmd/krel/cmd/announce_build.go

```diff
@@ -0,0 +1,291 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"text/template"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"k8s.io/release/pkg/command"
+)
+
+const (
+	branchFlag            = "branch"
+	changeLogFilePathFlag = "changelog-file-path"
+	changeLogFileNameFlag = "changelog-filename"
+	changeLogHTMLFlag     = "changelog-html-file"
+	workDirFlag           = "workdir"
+)
+
+const semVerRegex string = `?([0-9]+)(\.[0-9]+)?(\.[0-9]+)`
+
+const branchCreationMsg = `Kubernetes Community,
+<P>
+Kubernetes' {{ .Branch }} branch has been created.
+<P>
+The release owner will be sending updates on how to interact with this branch shortly.  The <A HREF=https://git.k8s.io/community/contributors/devel/sig-release/cherry-picks.md>Cherrypick Guide</A> has some general guidance on how things will proceed.
+<P>
+Announced by your <A HREF=https://git.k8s.io/sig-release/release-managers.md>Kubernetes Release Managers</A>.
+`
```

Comment:
I suggest refactoring the HTML, too. For example lowercase the tags and use `<p>` tags in a wrapped way: `<p>Text</p>. Add a `target="_blank"` to the links to open them in a new tab. :)

Reply:
done